### PR TITLE
Feature/chromecast events

### PR DESCRIPTION
**Pull request by ivo-toby**

### CODE REVIEW

**Comment by ivo-toby** [comment]
Use https://github.com/meisterplayer/meister-cast-target-template as target

**Comment by FranklinWaller** [comment]
Can't this be placed in the chromecast plugin? Because Meistercore doesn't really have anything to do with chromecast

**Comment by ivo-toby** [comment]
I thought these events should be defined here, like other plugins (like ads). In the end all events available are defined in one place (core)

File: src/js/events/chromecastreceiver-events.js [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+    'queueRemove',
+    'queueReorder',
+    'queueGetItemRange',
+    'queueGetItems',
+    'queueGetItemsIndex',
+    'castError',
+    'playerLoading',
+    'playerPreloading',
+    'playerPreloadingCancelled',
+    'canPlay',
```
Comment (Thomasvdam) [inline src/js/events/chromecastreceiver-events.js]: Shouldn't there be something in the event name to indicate this is exclusive to ChromeCast? I see several names that are ambiguous and could easily apply to several other plugins.

**Review by Thomasvdam — CHANGES_REQUESTED** [review]
I also agree with Franklin's comment that ideally plugin specific events shouldn't be declared in the core. We declared the ad events here with the idea that they could be universal among different ad providers.

Now that I'm thinking about it this entire section is rather pointless, as the only real function it has is to prevent a console.warn from displaying when firing an event that has no listeners. And I suppose it also serves as a lackluster reference of possible event names, but without any way to import/use the exact strings in other plugins it makes very little sense. This is something we should take a closer look at when cleaning up the event bus.

**Comment by Thomasvdam** [comment]
To clarify: I don't see placing these events here as a problem for this pull request, that's a new issue all by itself. But I do think that these events should be namespaced in some way to at least make them a little bit more meaningful. :)

**Comment by ivo-toby** [comment]
In meister6 we should rethink how events are exposed.. For now I renamed events to their own namespace

**Review by Thomasvdam — APPROVED** [review]

**Pull request merged**